packages/app/search/service.ts

```diff
@@ -0,0 +1,29 @@
+import compile from 'monquery'
+import type { Model } from '../models/types'
+import type { ErrorData } from '../declarations'
+import { searchModelsDb } from './db'
+import log from '../lib/log'
+
+/**this searvice takes lucene query string and converts it to mongoDB query*/
+
+export function searchwithMonquery(searchQuery) {
+    let search = compile(searchQuery)
```

Comment:
Could we add some error handling here? In the change that the searchQuery is mistyped, this would potentially throw a JS error

If we can catch that error and return an ErrorData response, similar to what getModel does, that would be awesome

Reply:
Sure, I will add error handling here.